lib/cc/config.rb

```diff
@@ -20,9 +20,9 @@ class Config
     def self.load
       config = Default.new
       if File.exist?(JSON::DEFAULT_PATH)
-        config = config.merge(JSON.new)
-      elsif File.exist?(YAML::DEFAULT_PATH)
-        config = config.merge(YAML.new)
+        config = config.merge(JSON.load)
+      elsif File.exist?(YAMLAdapter::DEFAULT_PATH)
+        config = config.merge(JSON.new(YAMLAdapter.load.config))
```

Comment:
Is there an interface difference here between `JSON.load` and `YAMLAdapter.load`? Wondering why we need the extra `#config` for the YAML adapater but not for the JSON adapter.

Reply:
Because the JSON hasn't been turned into an `Adapter` of the same style *yet*. It will be.